rust-toolchain

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.60.0"
```

Comment:
I would suggest:
```suggestion
channel = "stable"
components = ["clippy", "rustfmt"]
```

Because:
- The code is only built and tested on latest stable Rust
- CI verifies `clippy` and `rustfmt` formatting, so they are "needed" components for development.